pipelines/push-to-external-registry/push-to-external-registry.yaml

```diff
@@ -190,6 +190,10 @@ spec:
       runAfter:
         - apply-mapping
     - name: push-snapshot
+      when:
```

Comment:
When clause doesn't work as expected when the `echo "true" | tee $(results.mapped.path)`  is used in "apply-mapping" task.  Use `printf "true" |  tee $(results.mapped.path)` instead of it.  See [link](https://stackoverflow.com/questions/76425780/tekton-debugging-why-a-when-clause-is-evaluated-as-false)